x-pack/plugins/apm/public/components/app/correlations/utils/get_failure_correlation_impact_label.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FailureCorrelationImpactThreshold } from '../../../../../common/search_strategies/failure_correlations/types';
+import { FAILURE_CORRELATION_IMPACT_THRESHOLD } from '../../../../../common/search_strategies/failure_correlations/constants';
+
+export function getFailureCorrelationImpactLabel(
```

Comment:
Quick win? Would be great to cover this with jest tests as part of this PR.

Reply:
Added unit tests for this here https://github.com/elastic/kibana/pull/108441/commits/c6940c111c38916c26cb02fa63a76ab1f6b1a7de